test/unit/helpers/Base64Test.php

```diff
@@ -0,0 +1,84 @@
+
+    /**
+     * @dataProvider dataProvider
+     *
+     * @param $value
+     * @param bool $result
+     */
+    public function testBase64Encoding($value, bool $result): void
+    {
+        $this->assertEquals(Base64::isEncoded($value), $result);
+    }
+
+    /**
+     * @dataProvider dataProvider
+     *
+     * @param $value
+     * @param bool $result
+     */
+    public function testBase64EncodingImage($value, bool $result): void
+    {
+        $this->assertEquals(Base64::isEncodedImage($value), $result);
```

Comment:
`assertEquals()` accepts an expected value as a first argument.

```suggestion
        $this->assertEquals($result, Base64::isEncoded($value));
    }

    /**
     * @dataProvider dataProvider
     *
     * @param $value
     * @param bool $result
     */
    public function testBase64EncodingImage($value, bool $result): void
    {
        $this->assertEquals($result, Base64::isEncodedImage($value));
```

Reply:
You are right! Thanks for your comment! :)